godot_project/editor/rendering/rendering.gd

```diff
@@ -61,12 +64,16 @@ func snapshot_rebuild(in_structure_context: StructureContext) -> void:
 
 func initialize(in_workspace_context: WorkspaceContext) -> void:
 	if not enabled: return
+	var selection_layer_bit_enumerated_from_0: int = SELECTION_PREVIEW_LAYER_BIT - 1
```

Comment:
nit: this variable will cause an unused warning when assert is stripped out in Release, probably rename it to `_selection_layer_bit_enumerated_from_0`